### PR TITLE
test: set up spectron

### PR DESCRIPTION
**Pull request by pyphilia**
This PR uses `"spectron": "11.0.0"` because the version `11.1.0` contains breaking changes and throws an error for every test.

### CODE REVIEW

File: src/config/selectors.js [hunk 1]
```diff
@@ -1,2 +1,3 @@
 export const HOME_MENU_ITEM_ID = 'homeMenuItem';
 export const DRAWER_BUTTON_ID = 'drawerButton';
+export const QUIT_MENU_ITEM_ID = 'closeMenuItem';
```
Comment (juancarlosfarah) [inline src/config/selectors.js:3]: Maybe consistency between `quit` and `close`?

File: test/menu.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      app = await createApplication();
+    }),
+  );
+
+  afterEach(function () {
+    return closeApplication(app);
+  });
+
+  it(
+    'Displays correct navigation',
```
Comment (juancarlosfarah) [inline test/menu.test.js:24]: Try to keep `it` blocks starting with lower case, to differentiate from `describe` blocks. Not important, just a tip.

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks for adding this. Just some minor recommendations. Otherwise all good from my end. Check with @hagoptaminian for merging.

**Review by hagoptaminian — APPROVED** [review]
this looks good to me! ran `yarn mocha` locally and the tests are working

**Pull request merged**